lib/widgets/appBar.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+AppBar appBar(BuildContext context) {
+  return AppBar(
+    title: const Text(
+      "Profile",
+      style: TextStyle(
+          color: Colors.black, fontSize: 18, fontWeight: FontWeight.bold),
+    ),
+    centerTitle: true,
+    leading: IconButton(
+      onPressed: () {
+        Navigator.pop(context);
+      },
+      icon: const Icon(Icons.arrow_back),
+    ),
```

Comment:
This is unnecessary because flutter automatically apply back button when there is many stacked navigations.